internal/featuretests/v3/directresponsepolicy_test.go

```diff
@@ -0,0 +1,128 @@
+// Copyright Project Contour Authors
```

Comment:
This file needs to be formatted with `goimports` - `goimports -w internal/featuretests/v3/directresponsepolicy_test.go` should do it.